rc/lsp.kak

```diff
@@ -342,18 +342,18 @@ have_kakoune_feature_filtertext = ${kak_opt_lsp_have_kakoune_feature_filtertext}
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }}
 
-declare-option -hidden str lsp_completion_inserted_range
+declare-option -hidden str lsp_completion_inserted_ranges
```

Comment:
Why not just change the type to `str-list` and use `eval` when setting the option from `%val{hook_param}`.

Reply:
great catch, thanks!